tests/integration/mci/test_submission_with_errors.py

```diff
@@ -37,43 +37,6 @@ def test_submission_with_errors(self):
         self.assertRegexpMatches(content, "What are the dates of the sales period you are reporting for\?")
         self.assertRegexpMatches(content, ">Save and continue<")
 
-        # We fill in our answers
-        form_data = {
-            # Start Date
-            "6fd644b0-798e-4a58-a393-a438b32fe637-day": "01",
-            "6fd644b0-798e-4a58-a393-a438b32fe637-month": "4",
-            "6fd644b0-798e-4a58-a393-a438b32fe637-year": "2016",
-            # End Date
-            "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0-day": "30",
-            "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0-month": "04",
-            "06a6a4b7-6ce4-4687-879d-3443cd8e2ff0-year": "2016",
-            # Total Turnover
-            "e81adc6d-6fb0-4155-969c-d0d646f15345": "100000",
-            # User Action
-            "action[save_continue]": "Save &amp; Continue"
-        }
-
-        # We submit the form
-
-        resp = self.client.post(block_one_url, data=form_data, follow_redirects=False)
-        self.assertEquals(resp.status_code, 302)
-
-        # There are no validation errors
-        self.assertRegexpMatches(resp.headers['Location'], mci_test_urls.MCI_0205_SUMMARY_REGEX)
-
-        summary_url = resp.headers['Location']
-
-        resp = self.client.get(summary_url, follow_redirects=False)
-        self.assertEquals(resp.status_code, 200)
-
-        # We are on the review answers page
-        content = resp.get_data(True)
-        self.assertRegexpMatches(content, '<title>Summary</title>')
-        self.assertRegexpMatches(content, '>Monthly Business Survey - Retail Sales Index</')
-        self.assertRegexpMatches(content, '>Your responses<')
-        self.assertRegexpMatches(content, 'Please check carefully before submission')
-        self.assertRegexpMatches(content, '>Submit answers<')
-
```

Comment:
It was submitting valid answers the submitting invalid answers, but as we no longer save invalid answers the validation on submission didn't fail as the answers were still valid.
so if we don't submit valid answers first then the submission has errors so redirects back to the invalid page.